docs/topics/db/queries.txt

```diff
@@ -865,6 +865,15 @@ precede the definition of any keyword arguments. For example::
 
 ... would not be valid.
 
+Differently from :meth:`~django.db.models.query.QuerySet.get` method, the 
+:meth:`~django.db.models.query.QuerySet.get_or_create` method must be used 
```

Comment:
why it `must` be used with `filter` lookup function?
`Poll.objects.get_or_create(...)` is fine.
Isn't it better `As we can't set a concrete field value from an arbitrary condition, get_or_create must be used with keyword argument lookups only (which are exact match for a specific value)`